Add a footer

evilstreak opened the pull request:
Includes a first pass of styling to look roughly like the prototype, with a little simplification.

<img width="506" alt="screen shot 2015-12-01 at 19 42 36" src="https://cloud.githubusercontent.com/assets/74812/11511945/b3a1a5fe-9863-11e5-855d-8378bb735357.png">

<img width="1138" alt="screen shot 2015-12-01 at 19 42 30" src="https://cloud.githubusercontent.com/assets/74812/11511949/b562eb0a-9863-11e5-84cd-61cce3ec599c.png">

evilstreak committed:
Add a footer

Includes a first pass of styling to look roughly like the prototype,
with a little simplification.

evilstreak commented:
Do we need to say "except where otherwise stated"?

Demotive commented:
I do not know if we need the "unless..." suggest we drop it. Then we'll find out if we do :smile:

Demotive commented on _sass/global/layout.scss:
```diff
@@ -39,6 +39,32 @@ body {
+    color: $grey-3;
+  }
+
+  .footer-container {
```
I've been `@extend .block-container` what's your opinion?
I just prefer to have those declarations in one place for an easier life...

evilstreak replied:
Ah yeah, I tried that but got the syntax wrong. Do you mean `.block-container`?

I'll update it to do that for both header and footer

Demotive replied:
_Obviously_ I mean `.block-container` :P

evilstreak replied:
Updated!

Demotive replied:
:+1:

evilstreak committed:
Extend .block-container to DRY up the CSS

We want the visuals of the header, content and footer to be the same, so
extend the same container class with the `max-width`, `padding` and
`margin`.

evilstreak committed:
Make HTML background match footer

Give the HTML element a black background so the footer visually extends
to the bottom of the page instead of floating.
